library/cache/src/main/java/com/gtc/getcamp/cache/LocalCacheProvider.kt

```diff
@@ -0,0 +1,10 @@
+package com.gtc.getcamp.cache
+
+import androidx.datastore.preferences.core.Preferences
+
+interface LocalCacheProvider {
+
+    suspend fun <T> storeValue(key: Preferences.Key<T>, value: T)
```

Comment:
@logicandroid: read yazılınca niye write yerine store yazmışsın

Reply:
It make sense.